com/win32comext/axscript/test/testHost.py

```diff
@@ -134,7 +134,7 @@ def testcollection():
      test.fail("Didn't get the collection")
    pass
 """
-
+# TODO
 # XXX - needs py3k work!  Throwing a bytes string with an extended char
 # doesn't make much sense, but py2x allows it.  What it gets upset with
 # is a real unicode arg - which is the only thing py3k allows!
```

Comment:
Not entirely certain if it's the test or just the message that should be updated here.

Reply:
I think killing the comment and references to `PyScript_Exc` seems fine.

Reply:
Do you mean that removing references to `PyScript_Exc` (and by extension, removing the `testPythonExceptions` test) is fine, or did you mean that having the references to `PyScript_Exc` is fine?

Left it for now, updated comment to mention Python 2 in the past tense.

Reply:
I think removing `PyScript_Exc` and that test is probably best.